if athena database name needs to be quoted, querying doesn't work

Hi, I can create a virtual schema, but I cannot query it.
`CREATE VIRTUAL SCHEMA VS_ufos
    USING ADAPTER.ATHENA_JDBC_ADAPTER
    WITH
    SQL_DIALECT = 'ATHENA'
    CONNECTION_NAME = 'FS_UFOS'
    SCHEMA_NAME = 'fs-eu-west-1';
    `
Error:
`An error occurred while performing the operation:
VM error: F-UDF-CL-LIB-1126: F-UDF-CL-SL-JAVA-1006: F-UDF-CL-SL-JAVA-1026:
com.exasol.ExaUDFException: F-UDF-CL-SL-JAVA-1068: Exception during singleCall adapterCall
java.lang.AssertionError: E-ID-2: Unable to create identifier "fs-eu-west-1" because it contains illegal characters. For information about valid identifiers, please refer to https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html
com.exasol.adapter.dialects.athena.AthenaIdentifier.of(AthenaIdentifier.java:49)
com.exasol.adapter.dialects.athena.AthenaSqlDialect.applyQuote(AthenaSqlDialect.java:112)
com.exasol.adapter.dialects.SqlGenerationVisitor.visit(SqlGenerationVisitor.java:184)
com.exasol.adapter.dialects.SqlGenerationVisitor.visit(SqlGenerationVisitor.java:30)
com.exasol.adapter.sql.SqlTable.accept(SqlTable.java:46)
com.exasol.adapter.dialects.SqlGenerationVisitor.visit(SqlGenerationVisitor.java:78)
com.exasol.adapter.dialects.SqlGenerationVisitor.visit(SqlGenerationVisitor.java:30)
com.exasol.adapter.sql.SqlStatementSelect.accept(SqlStatementSelect.java:104)
com.exasol.adapter.dialects.AbstractQueryRewriter.createPushdownQuery(AbstractQueryRewriter.java:62)
com.exasol.adapter.dialects.AbstractQueryRewriter.rewrite(AbstractQueryRewriter.java:48)
com.exasol.adapter.dialects.AbstractSqlDialect.rewriteQuery(AbstractSqlDialect.java:134)
com.exasol.adapter.jdbc.JdbcAdapter.pushdown(JdbcAdapter.java:210)
com.exasol.adapter.RequestDispatcher.dispatchPushDownRequestToAdapter(RequestDispatcher.java:142)
com.exasol.adapter.RequestDispatcher.processRequest(RequestDispatcher.java:80)
com.exasol.adapter.RequestDispatcher.executeAdapterCall(RequestDispatcher.java:52)
com.exasol.adapter.RequestDispatcher.adapterCall(RequestDispatcher.java:41)
com.exasol.ExaWrapper.runSingleCall(ExaWrapper.java:100)
(Session: 1703537717580726272)`

It is probably due to the minus characters in the schema_name. I think the adapter would have to put it in backticks as described here:
https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html
Thank you for investigating.

--- a/doc/changes/changes_2.0.0.md
+++ b/doc/changes/changes_2.0.0.md
@@ -1,4 +1,4 @@
-# Virtual Schema for Athena 2.0.0, released 2021-??-??
+# Virtual Schema for Athena 2.0.0, released 2021-08-02
 
 Code name: Removed `SQL_DIALECT` property
 
@@ -9,6 +9,8 @@ The `SQL_DIALECT` property used when executing a `CREATE VIRTUAL SCHEMA` from th
 ## Documentation
 
 * #4: Described String data type.
+* #6: Added troubleshooting information about working groups.
+* #7: Added troubleshooting information about allowed schema names in Athena.
 * #8: Improved user guide, replaced outdated information.
 * #11: Added a troubleshooting guide.
 

--- a/doc/user_guide/athena_user_guide.md
+++ b/doc/user_guide/athena_user_guide.md
@@ -119,7 +119,7 @@ In this example we used the maximum length of the Exasol Varchar datatype.
 
 ## Troubleshooting 
 
-## SELECT Query Hangs and Returns Timeout
+### SELECT Query Hangs and Returns Timeout
 
 If you created a Virtual Schema successfully, but a SELECT query runs forever without any result, check the following things:
 
@@ -154,3 +154,13 @@ If you created a Virtual Schema successfully, but a SELECT query runs forever wi
    You can find the logs in the `/tmp/athena/` directory on the Exasol Node.
 
 * See also: https://aws.amazon.com/premiumsupport/knowledge-center/athena-connection-timeout-jdbc-odbc-driver/
+
+### You get an error: Value null at 'workGroup' failed to satisfy constraint: Member must not be null
+
+**Solution**: use the JDBC driver 2.0.23 or later. Some old driver versions had a bug leading to this error.
+See the driver's [changelog](https://s3.cn-north-1.amazonaws.com.cn/athena-downloads-cn/drivers/JDBC/SimbaAthenaJDBC-2.0.23.1000/docs/release-notes.txt) file for additional information.
+
+### If Athena database/table/column name has special characters except underscore, Virtual Schema throws an error
+
+**Solution**: This is an expected behavior. We validate on the Virtual Schema side that Athena identifiers only contain supported characters following this rule: `Special characters other than underscore (_) are not supported.`
+See [the official AWS documentation](https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html) for more information.

